docs/verifiable_credentials_examples/talent_protocol_passports/panagiotis_matsinopoulos.json

```diff
@@ -0,0 +1,63 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "https://passport.talentprotocol.com/passports/0x324e9E13dd19528D0F390201923d17c4B7E94462",
```

Comment:
here we should use id instead of wallet since the wallet might change but the id doesn’t